docs/topics/kotlin-nights-branding.md

```diff
@@ -1,4 +1,4 @@
-[//]: # (title: Kotlin Nights branding)
+[//]: # (title: Kotlin Nights Guidelines)
```

Comment:
@nikpachoo can we please have Kotlin Night Guidelines instead?

Reply:
We don't use title case at kotlinlang.org/docs